Quarkus continuous test mode hides server output

### Describe the bug

I am using the new continuous test mode, which is awesome.
Now when I work on some test-driven coding, I write a test, write the code and then Quarkus runs my test and shows me that my code is not working:

```
2021-07-21 12:26:16,957 ERROR [io.qua.test] (Test runner thread) ==================== TEST REPORT #11 ====================
2021-07-21 12:26:16,957 ERROR [io.qua.test] (Test runner thread) Test CheckerTest#basicCheck() failed 
: java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <500>.

        at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
        at com.redhat.clouddot.poc.funq.CheckerTest.basicCheck(CheckerTest.java:26)

2021-07-21 12:26:16,957 ERROR [io.qua.test] (Test runner thread) >>>>>>>>>>>>>>>>>>>> 1 TEST FAILED <<<<<<<<<<<<<<<<<<<<


--
1 test failed (1 passing, 0 skipped), 2 tests were run in 467ms. Tests completed at 12:26:16.
Press [r] to re-run, [v] to view full results, [p] to pause, [h] for more options>
```

But it hides the internal error message that the server would produce, which would help me understand why my code fails.

E.g. when I trigger the same rest-test not from a `@QuarkusTest` and `RestAssured`, but via `curl` , I get my normal stack trace that tells me what does not work in my code.


### Expected behavior

Test failure output and server output are shown

### Actual behavior

Only test output is shown, but server output is omitted

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.1.0.CR1

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java
+++ b/core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java
@@ -33,7 +33,8 @@ public class TestConsoleHandler implements TestListener {
 
     private static final Logger log = Logger.getLogger("io.quarkus.test");
 
-    public static final ConsoleCommand TOGGLE_TEST_OUTPUT = new ConsoleCommand('o', "Toggle test output",
+    public static final ConsoleCommand TOGGLE_TEST_OUTPUT = new ConsoleCommand('o', "Toggle test output", "Toggle test output",
+            1000,
             new ConsoleCommand.HelpState(TestSupport.instance().get()::isDisplayTestOutput),
             () -> TestSupport.instance().get().toggleTestOutput());
 

